content/Tjenester/fiksprotokoll/brukerveiledning/huskeliste_for_opprette_protokoll.md

```diff
@@ -27,13 +27,16 @@ Se huskelisten lenger nede for hvilke ressurser det gjelder.
 ## Huskeliste
 
 ### 1. Maskinporten
-Leverandør(ene) skal ha satt opp integrasjon med Maskinporten med **sitt** organsiasasjonsnummer på forhånd. Dette organsiasjonsnummeret skal brukes i oppsett av *"system"*
+Leverandør(ene) skal ha satt opp integrasjon med Maskinporten på forhånd. Her kan man velge å enten bruke leverandøren sitt organisasjonsnummer eller kunden (kommunen) sitt organisasjonsnummer pluss tilhørende virksomhetssertifikat. 
+Det samme organsiasjonsnummeret **må** brukes i oppsett av *"system"*. 
+
+Organisasjonsnummer skal være likt i maskinportenintegrasjon, *"system"* i FIKS forvaltning og virksomhetssertifikatet som brukes i maskinporten integrasjonen (og dermed også FIKS IO klienten).
```

Comment:
Ble usikker på linje 33, der man nevner maskinporten-integrasjon to ganger.
Kan man kanskje skrive noe sånt:

Leverandør(ene) skal ha satt opp integrasjon med Maskinporten på forhånd. Her kan man enten velge å bruke leverandøren sitt organisasjonsnummer + virksomhetssertifikat eller kunden (kommunen) sitt organisasjonsnummer + virksomhetssertifikat. 

Merk: det organisasjonsnummer man velger å bruke **må** også brukes i oppsett av *"system"* på Fiks forvaltning.

Reply:
Vet ikke om det ble så mye bedre 🤔 Man gjør feil her ofte så jeg tenker at det er greit å til slutt poengtere at det MÅ være likt orgnr i de 3. Altså maskinportenintegrasjonen, systemet i FIKS forvaltning og virksomhetssertifikatet.

Reply:
Nytt forslag lagt inn